_data/changelogs/component-card.yml

```diff
@@ -2,6 +2,14 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused the component to ignore the `$theme-card-font-family` setting.
```

Comment:
**suggestion (non-blocking)**: Since this is marked as breaking change, we should include  details about the break users should look for. 

```suggestion
    summary: Fixed a bug that caused the component to ignore the `$theme-card-font-family` setting.
    summaryAdditional: Confirm that your implementation of the card component displays with the expected font family. 
```